presto-testing/src/test/java/io/prestosql/testing/datatype/TestDataType.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.testing.datatype;
+
+import org.testng.annotations.Test;
+
+import java.time.LocalTime;
+
+import static io.prestosql.testing.datatype.DataType.timeDataType;
+import static org.testng.Assert.assertEquals;
+
+public class TestDataType
+{
+    @Test
+    public void testTimeDataType()
```

Comment:
This seems unrelated to supporting `time(p)` in postgresql

Reply:
it was valuable to me, as i initially forgot to add SSSSSS to the pattern. I can throw it away if you want me, but otherwise I would keep it.

(note that "`time(p)` in postgresql" commit changes DataType.timeDataType... that can be seen frivolous, but postgres's tests were the only user of that method)

Reply:
Fine to keep, but move to a separate commit.

Reply:
It is in the commit which changes the impl of the feature under test. I cannot move to separate, that would not be logical separation.
Since this seems like a tiny detail, i will choose to keep it where it is.

Reply:
Ah, I missed that the `timeDataType` method was being changed in this same commit.